contracts/voting-verifier/src/events.rs

```diff
@@ -635,6 +494,7 @@ mod test {
                 None,
             ],
         }
+        .non_generic()
         .into();
 
         goldie::assert_json!(json!({
```

Comment:
can you make sure that we're exhaustively testing all events? So the test should break if we add a new event variant without adding it here

Reply:
Not sure how you can do that, but it's not necessary since this test is there to make sure the differences in the JSON before and after using the macro are OK. With new events, do we need them to be added here?